blueprint/index.md

```diff
@@ -0,0 +1,311 @@
+---
+title: Conversation model to SQL Blueprint
+author: ronan.watkins
+indextype: blueprint
+icon: blueprint
+image: images/erdiagram.png
+category: 6
+summary: |
+  This Genesys Blueprint provides an example of how to design an SQL database for storing JSON data, specifically from the [POST 
+  /api/v2/analytics/conversations/details/query](https://developer.genesys.cloud/api/rest/v2/analytics/#post-api-v2-analytics-conversations-details-query) endpoint. A project containing a dockerized postgres database and Node.js Typescript driver will be included as an example.
```

Comment:
There's a newline in the link, I'm not sure if that will display correctly.

Don't include the dev center domain in any links in the blueprint content pages; there are several instances I noticed in this page. Full URLs are necessary in the main README.md though since that's meant to be viewed on github.